docs/cloudfoundry/config_examples/manifest.yaml

```diff
@@ -66,25 +66,40 @@ applications:
 
       # Include the section below to use the cf-bigip-ctlr as a Service Broker
       # THE SETTINGS IN THIS SECTION ARE ROUTE-SPECIFIC
+      # The first health monitor references a monitor that already exists in the cf partition on the BIG-IP device
```

Comment:
Does it have to exist in the cf partition? Or can it be in /Common?

Reply:
It has to be in /Common. Unless it's one we're creating CCCL should end up deleting a pre-created monitor in the managed partition.

Reply:
When referencing a health monitor it can be in whatever partition you want as long as already exists on the BIG-IP before we create objects that reference it.

Reply:
Ya I fixed that reference in the example YAML.

Reply:
The comment should be updated too and reference common instead of the cf partition.

Reply:
Fixed.